### PR TITLE
Hackathon h1014 l4 wallets

### PR DESCRIPTION
**Pull request by vsrivatsa25**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## Description
Added wallet support for Bluesnap

### Additional Changes

- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## Checklist

- [ ] I formatted the code `cargo +nightly fmt --all`
- [ ] I addressed lints thrown by `cargo clippy`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

File: config/Development.toml [hunk 1]
```diff
@@ -43,7 +43,7 @@ locker_decryption_key2 = ""
```
Comment (SamraatBansal) [inline config/Development.toml:43]: I have added the reviews for most of the files in this PR in L1-L3 PR (PR: #533 ).

Please attend to those changes.

File: crates/router/src/connector/bluesnap/transformers.rs [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+                } else {
+                    "automatic".to_string()
+                };
+                let auth_mode = if capture_method.to_lowercase() == "manual" {
+                    BluesnapPaymentStatus::AuthOnly
+                } else {
+                    BluesnapPaymentStatus::AuthCapture
+                };
+                let payment_request = Self {
+                    wallet_id: Some("21".to_string()),
```
Comment (SamraatBansal) [inline crates/router/src/connector/bluesnap/transformers.rs:129]: Why is this `wallet_id` hardcoded?

Comment (SamraatBansal) [inline crates/router/src/connector/bluesnap/transformers.rs:129]: No need to write `payment_request` again, you can see other connectors to understand how to use the same struct with different values.

**Review by SamraatBansal — CHANGES_REQUESTED** [review]
Please attend to comments raised in previous PR also.

**Comment by jagan-jaya** [comment]
Closing this PR in favour of #649

**Pull request closed without merging**